### PR TITLE
Maint/replace edb with pyedb

### PR DESCRIPTION
**Pull request by maxcapodi78**

### CODE REVIEW

File: doc/source/User_guide/intro.rst [hunk 1]
```diff
@@ -32,3 +32,17 @@ This code creates a project and saves it with PyAEDT:
     ...
     cir.release_desktop(save_project=True, close_desktop=True)
     # Desktop is released here.
+
+Ansys EDB proprietary layout format is accessible through pyaedt using the following
```
Comment (PipKat) [inline doc/source/User_guide/intro.rst:36]:
```suggestion
The Ansys EDB proprietary layout format is accessible through PyAEDT using this
```

File: examples/00-EDB/Readme.txt [hunk 2]
```diff
@@ -1,5 +1,19 @@
 EDB examples
 ~~~~~~~~~~~~
-These examples use EDB (Electronics Database) with PyAEDT.
 EDB is a powerful API that allows to control PCB data efficently.
 You can either use EDB standalone or embedded in HFSS 3D Layout in AEDT.
+EDB class in now part of PyEDB package which is currently installed with PyAEDT and back compatible with PyAEDT.
+All EDB related examples have been moved
+at  `PyEDB examples page <https://edb.docs.pyansys.com/version/stable/examples/index.html>`_.
+These examples use EDB (Electronics Database) with PyAEDT.
```
Comment (PipKat) [inline examples/00-EDB/Readme.txt:8]:
```suggestion
EDB is a powerful API for efficiently controlling PCB data.
You can either use EDB standalone or embedded in HFSS 3D Layout in AEDT.
The ``EDB`` class in now part of the PyEDB package, which is currently installed with PyAEDT and backward-compatible with PyAEDT.
All EDB related examples have been moved
to the `Examples page <https://edb.docs.pyansys.com/version/stable/examples/index.html>`_ in the PyEDB
documentation.
These examples use EDB (Electronics Database) with PyAEDT.
```

File: doc/source/User_guide/intro.rst [hunk 3]
```diff
@@ -32,3 +32,17 @@ This code creates a project and saves it with PyAEDT:
...
+code:
+
+.. code:: python
+
+    # Launch the latest installed version of AEDB.
+    import pyaedt
+    edb = pyaedt.Edb("mylayout.aedb")
+
+    # User can launch Edb directly from PyEDB class.
```
Comment (PipKat) [inline doc/source/User_guide/intro.rst:45]:
```suggestion
    # You can also launch EDB directly from PyEDB.
```

File: examples/00-EDB/Readme.txt [hunk 4]
```diff
@@ -1,5 +1,19 @@
...
+
+.. code:: python
+
+    # Launch the latest installed version of AEDB.
+    import pyaedt
+    edb = pyaedt.Edb("mylayout.aedb")
+
+    # User can launch Edb directly from PyEDB class.
```
Comment (PipKat) [inline examples/00-EDB/Readme.txt:16]:
```suggestion
    # You can also launch EDB directly from PyEDB.

File: examples/00-EDB/Readme.txt [hunk 5]
```diff
@@ -1,5 +1,19 @@
...
 EDB is a powerful API that allows to control PCB data efficently.
 You can either use EDB standalone or embedded in HFSS 3D Layout in AEDT.
+EDB class in now part of PyEDB package which is currently installed with PyAEDT and back compatible with PyAEDT.
+All EDB related examples have been moved
+at  `PyEDB examples page <https://edb.docs.pyansys.com/version/stable/examples/index.html>`_.
+These examples use EDB (Electronics Database) with PyAEDT.
+
+.. code:: python
+
+    # Launch the latest installed version of AEDB.
```
Comment (PipKat) [inline examples/00-EDB/Readme.txt:12]: Possibly change to EDB?

File: examples/03-Maxwell/Maxwell2D_DCConduction.py [hunk 6]
```diff
@@ -69,7 +69,7 @@
 # Voltage ports will be defined as perfect electric conductor (pec), conductor
```
Comment (PipKat) [inline examples/03-Maxwell/Maxwell2D_DCConduction.py:69]:
```suggestion
# A voltage port is defined as a perfect electric conductor (pec). A conductor
```

File: examples/03-Maxwell/Maxwell2D_DCConduction.py [hunk 7]
```diff
@@ -69,7 +69,7 @@
...
 # gets the material defined by the 0th entry of the material array
```
Comment (PipKat) [inline examples/03-Maxwell/Maxwell2D_DCConduction.py:70]:
```suggestion
# gets the material defined by the 0th entry of the material array.
```

File: pyaedt/modeler/modelerpcb.py [hunk 8]
```diff
@@ -92,17 +92,23 @@ def _edb_file(self):
...
+        """EBD. Supported only in Ironpython
 
         Returns
         -------
         :class:`pyaedt.edb.Edb`
              EDB.
 
         """
+        if is_ironpython:
+            self.logger.warning("Edb is supported only in CPython.")
```
Comment (PipKat) [inline pyaedt/modeler/modelerpcb.py:104]:
```suggestion
            self.logger.warning("EDB is supported only in CPython.")
```

**Review by Samuelopez-ansys — APPROVED** [review]
LGTM!

**Review by SMoraisAnsys — APPROVED** [review]
LGTM

File: pyproject.toml [hunk 9]
```diff
@@ -35,7 +35,8 @@ dependencies = [
     "dotnetcore2 ==3.1.23;platform_system=='Linux'",
     "fpdf2",
     "jsonschema",
-    "pytomlpp"
+    "pytomlpp; python_version < '3.12'",
+    "pyedb"
```
Comment (SMoraisAnsys) [inline pyproject.toml:39]:
```suggestion
    "pyedb==0.2.0"
```

**Review by Samuelopez-ansys — APPROVED** [review]

**Pull request merged**